Automattic/jetpack
# Required Review: remove rules except for Sync, Connection, monorepo

- Remove unmatched rule
- Allow everyone ability to edit docs
- Reserve Sync and Conneciton packages
- Allow all to change changelogs, etc
- Remove all team-specific rules





As part of flipping the script on the review process, allow all a12s to review anything in the repo with the exception of Sync and Connection packages, plus the monorepo tooling itself.

## Proposed changes:

* See above.

### Other information:

n/a

## Jetpack product discussion


pdWQjU-fR-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* See if the Required Review check passes when someone not in the Approvers group approves anything outside of the reserved areas mentioned.
